cmd/modern/root/query.go

```diff
@@ -63,11 +63,12 @@ func (c *Query) run() {
 		line = console.NewConsole("")
 		defer line.Close()
 	}
-	s := mssql.Connect(endpoint, user, line)
+	sql := mssql.New(false)
```

Comment:
>ew(false) [](http://example.com/codeflow?start=15&length=9)

Boolean parameters are where I miss named parameters in C#. I have to look up the New function to figure out what is being set to false.

Reply:
Good point, in my IDE (Goland), I see:

	sql := mssql.New( unitTesting: false)
	
The IDE puts the variable name in the display.  But for code reviews and other IDEs that don't do this, it is not clear at all.

I'll move this to the Options pattern which is idiomatic for golang

Reply:
Moved to an Options parameter in:

[https://github.com/microsoft/go-sqlcmd/pull/205](https://github.com/microsoft/go-sqlcmd/pull/205)